holoviews/core/ndmapping.py

```diff
@@ -486,6 +486,7 @@ def info(self):
                 if d.value_format:
                     dmin, dmax = d.value_format(dmin), d.value_format(dmax)
                 info_str += '\t %s: %s...%s \n' % (d.pprint_label, dmin, dmax)
+        print(info_str)
```

Comment:
I know the docstring says "print", but I think what was actually intended is "return", which then prints in Jupyter or on the commandline but can also be used in other ways:

```suggestion
        return info_str
```

My guess is that `return info_str` was deleted accidentally at some point, and you're the first to notice, so thanks for pointing it out!